### PR TITLE
Fixes made in nalanda-rct3 and then brought into the central server submodule

### PR DESCRIPTION
**Pull request by jamalex**
Merging back to develop, unless anything looks dangerous, so that we avoid future merge conflicts/divergence.

### CODE REVIEW

**Comment by jamalex** [comment]
(This doesn't include the latest nalanda changes, just the ones as of a couple of weeks ago, which were needed in order to support Nalanda reports etc on the central server)

**Comment by jamalex** [comment]
@rtibbles Would you be able to have a peek?

File: kalite/settings.py [hunk 1]
```diff
@@ -169,10 +169,6 @@ def package_selected(package_name):
 
 import_installed_app_settings(INSTALLED_APPS, globals())
 
-KHAN_EXERCISES_RELPATH = os.path.join("static", "perseus", "ke")
```
Comment (rtibbles) [inline kalite/settings.py]: Where did these go?

File: kalite/topic_tools/settings.py [hunk 2]
```diff
@@ -11,4 +12,8 @@
 #######################
 # Set module settings
 #######################
-DO_NOT_RELOAD_CONTENT_CACHE_AT_STARTUP = getattr(local_settings, "DO_NOT_RELOAD_CONTENT_CACHE_AT_STARTUP", False)
\ No newline at end of file
+DO_NOT_RELOAD_CONTENT_CACHE_AT_STARTUP = getattr(local_settings, "DO_NOT_RELOAD_CONTENT_CACHE_AT_STARTUP", False)
+
+KHAN_EXERCISES_RELPATH = os.path.join("static", "perseus", "ke")
```
Comment (rtibbles) [inline kalite/topic_tools/settings.py]: Ah, here it is.

**Comment by rtibbles** [comment]
Nothing jumps out. Tests pass. Merging.

**Comment by jamalex** [comment]
Yayyy, thanks!

**Pull request merged**